Gaijinizer patch/Map037.txt

```diff
@@ -1106,9 +1134,9 @@ In your 'will as a person'.#MTLed
 
 私はこれにて…
 > CONTEXT: Map037/events/4/pages/1/519/Dialogue/yuuma
-Well, maintain the substance too much
-I'm not going to break into a grotesque ...
-I am at this ...#MTLed
+Well then, I won't try to maintain this form much
+longer, I won't go and grotesquely collapse...
+Like this I'll...#RonIffy
 > END STRING
```

Comment:
This is definitely a hard one, but here's probably a better go at it:
>_**Well then, I won't try to maintain this 
form much longer. I'm not going to let myself gruesomely
fall to bits in front of you...
I- I'm...**_